packages/sdks/src/components/blocks/blocks.types.ts

```diff
@@ -15,4 +15,5 @@ export type BlocksProps = Partial<
   registeredComponents?: RegisteredComponents;
   linkComponent?: any;
   className?: string;
+  extraAttributesForBlocksWrapper?: Record<string, any>;
```

Comment:
discussed in slack about how to handle these extra attrs. we want to avoid the extra prop and instead rely on BlocksWrapperProps


- `Blocks` should now receive `BlocksWrapperProps`
- if its provided by user, override the context value completely

Reply:
and lets make sure to properly document this behaviour in JSDoc and the changeset